torch/nn/functional.py

```diff
@@ -632,18 +637,17 @@ def max_unpool3d(input, indices, kernel_size, stride=None, padding=0,
 
 @weak_script
 def lp_pool2d(input, norm_type, kernel_size, stride=None, ceil_mode=False):
-    # type: (Tensor, float, int, Optional[BroadcastingList2[int]], bool) -> Tensor
+    # type: (Tensor, float, BroadcastingList2[int], Optional[BroadcastingList2[int]], bool) -> Tensor
     r"""Applies a 2D power-average pooling over an input signal composed of
     several input planes. If the sum of all inputs to the power of `p` is
     zero, the gradient is set to zero as well.
 
     See :class:`~torch.nn.LPPool2d` for details.
     """
-    kw, kh = utils._pair(kernel_size)
-    if stride is not None:
-        out = avg_pool2d(input.pow(norm_type), kernel_size, stride, 0, ceil_mode)
-    else:
-        out = avg_pool2d(input.pow(norm_type), kernel_size, padding=0, ceil_mode=ceil_mode)
+    kw, kh = _pair(kernel_size)
+    if stride is None:
+        stride = torch.jit.annotate(List[int], kernel_size)
```

Comment:
Modest question: Why did we annotate this as a `List[int]` instead of a `BroadcastingList2[int]`?

Reply:
I tried :P but it turns out `BroadcastingList2[int]` is not a valid option in `torch.jit.annotate`. talked to @driazati about it, it's mainly because `BroadcastingList<x>` is a "hacky" workaround and we only supports `x` up to `5`.